Sources/Nimble/DSL+Wait.swift

```diff
@@ -89,11 +89,17 @@ internal class NMBWait: NSObject {
 
 #if (os(macOS) || os(iOS) || os(tvOS) || os(watchOS)) && !SWIFT_PACKAGE
     @objc(untilFile:line:action:)
-    internal class func until(_ file: FileString = #file, line: UInt = #line, action: @escaping (() -> Void) -> Void) {
+    internal class func until(
+        _ file: FileString = #file,
+        line: UInt = #line,
+        action: @escaping (@escaping () -> Void) -> Void) {
```

Comment:
This now matches

https://github.com/Quick/Nimble/blob/3f25b677eaadd009987386a368de173d9c0cad93/Sources/Nimble/DSL%2BWait.swift#L18-L23

Completely makes sense 👍